spec/features/comment_posts_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+require 'create_user_and_sign_up'
+
+RSpec.feature "Comment", type: :feature do
+  scenario "User can comment on a post" do
+    create_user_and_sign_up
+    click_link 'New post'
+    fill_in 'post_caption', with: 'Cute cat'
+    page.attach_file('post_image', Rails.root + "spec/fixtures/stanley.jpg")
+    click_button 'Create post'
+    fill_in 'comment_body', with: 'Your cat is super cute'
+    click_button 'Add'
+    expect(page).to have_content('Your cat is super cute')
```

Comment:
the review guidelines suggest making test like this more specific by checking that the image is in the right place using `within`, e.g.:

```
within("comment#photo6") do
  expect(page).to have_content 'Awesome Photo!'
end
```

I guess you have to strike a balance between being specific and the test being too tightly coupled to the views layer though.

https://github.com/Hives/instagram-challenge/blob/master/docs/review.md#feature-test-precision-using-within